app/Models/AuthorizedUser.php

```diff
@@ -16,19 +16,22 @@
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
 use Illuminate\Support\Collection;
-use Laravel\Sanctum\HasApiTokens;
 use Spatie\Permission\Traits\HasRoles;
 
-class User extends Authenticatable implements CanResetPasswordContact
+class AuthorizedUser extends Authenticatable implements CanResetPasswordContact
 {
-    use HasApiTokens;
     use HasFactory;
     use Notifiable;
     use SoftDeletes;
     use HasUuid;
     use HasRoles;
     use CanResetPassword;
 
+    public const AUTHENTICATION_GUARD = 'authorized_users';
+    public const AUTHENTICATION_PROVIDER = 'authorized_users';
+
+    public const DATABASE_TABLE = 'authorized_users';
```

Comment:
Is dit nodig? Indien je een andere naam voor de tabel kan je het volgende gebruiken;
```php
protected $table = 'authorized_users';
```

En om de tabel naam op te halen kan je dit volgens mij dit het volgende doen.
```php
$authorizedUser = new AuthorizedUser();
AuthorizedUser->getTable();

// Short
(new AuthorizedUser())->getTable();
```

Denk dat dit handiger is om te gebruiken in de context waarvoor het nodig is.

Reply:
Good one. I will change it.